### PR TITLE
Prepared data for contractors frontend index page and replaced dummy

### PR DESCRIPTION
**Pull request by AironDev**

### CODE REVIEW

**Comment by EphraimDev** [comment]
Is this still wip?

File: resources/views/pages/contract/index.blade.php [hunk 1]
```diff
@@ -10,11 +10,11 @@
 	<section>
       <div class="container">
       {{ Breadcrumbs::render('contractors') }}
-        <h1 class="ws-10 font-weight-bold">Contracted Companies and Organisations</h1>
+        <h1 class="ws-10 font-weight-bold">Contracted contractors and Organisations</h1>
```
Comment (goodhands) [inline resources/views/pages/contract/index.blade.php]: I think this should be "Companies", "Contracted contractors" sounds like repetition.

File: resources/views/pages/contract/index.blade.php [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
         <br />
         <div class="row">
             <div class="col-md-5">
-                <p class="para">ExpenseNG gives an insight to how much is being dispensed to contracted companies.</p>
+                <p class="para">ExpenseNG gives an insight to how much is being dispensed to contracted contractors.</p>
```
Comment (goodhands) [inline resources/views/pages/contract/index.blade.php]: same here

File: resources/views/pages/contract/index.blade.php [hunk 3]
```diff
@@ -40,27 +40,27 @@
...
 
     <br />
-    <div id="company" class="container">
-    <div class="row" id="company-div">
-        @foreach ($companies as $company)  
-          <a href="{{ route('contractors.single', ['company' => $company->shortname()]) }}">
+    <div id="contractor" class="container">
+    <div class="row" id="contractor-div">
+        @foreach ($contractors as $contractor)  
+          <a href="#">
```
Comment (goodhands) [inline resources/views/pages/contract/index.blade.php]: we should link to the single page for each contractor right?

File: resources/views/pages/contract/index.blade.php [hunk 4]
```diff
@@ -40,27 +40,27 @@
...
-                            element="{{ $company->shortname() }}"></chart>
-                    <div class="company mb-2">
-                        <img src="{{ asset('images/image 13.png') }}" height="30" class="mr-3" alt="">
-                        <a href="{{ route('contractors.single', ['company' => $company->shortname()]) }}">
-                          <h5 class="card-title mb-0" class="company-name">
-                            {{ $company->name }}
+                            v-bind:data="[{amount: {{round($contractor->total_amount / 12)}}, year: {{$contractor->year}} }, {amount:{{$contractor->total_amount}}, year:{{$contractor->year}} }]"
+                            element="{{ \Str::slug($contractor->beneficiary, '-') }}"></chart>
+                    <div class="contractor mb-2">
+                        <!-- <img src="{{ asset('images/image 13.png') }}" height="30" class="mr-3" alt=""> -->
```
Comment (goodhands) [inline resources/views/pages/contract/index.blade.php:54]: I'm thinking in future, we should have a db column for accepting the logo url for a company. Same with ministries. This should come in later.

File: resources/views/pages/contract/index.blade.php (hunk 1 above)
Comment (AironDev) [reply resources/views/pages/contract/index.blade.php]: Noted

File: resources/views/pages/contract/index.blade.php (hunk 3 above)
Comment (AironDev) [reply resources/views/pages/contract/index.blade.php]: Yea - I have handled that already, will push a commit

**Comment by goodhands** [comment]
nice, thanks



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
07/29/20,
08:27:02 PM

**Comment by AironDev** [comment]
@EphraimDev  done

**Pull request merged**